darshan-util/pydarshan/darshan/experimental/plots/heatmap_handling.py

```diff
@@ -273,7 +273,7 @@ def get_aggregate_data(
     return agg_df
 
 
-def get_heatmap_df(agg_df: pd.DataFrame, xbins: int) -> pd.DataFrame:
+def get_heatmap_df(agg_df: pd.DataFrame, xbins: int, nprocs: int) -> pd.DataFrame:
```

Comment:
when you change a function signature in the main library, it is important to grep the codebase for other places it is used like the `asv` benchmarks, which this breaks..

I'll try to fix this now

Reply:
Ok, I pushed in the fix and rebased as well